man/runc-features.8.md

```diff
@@ -0,0 +1,28 @@
+% runc-features "8"
+
+# NAME
+**runc-features** - show implemented features
+
+# SYNOPSIS
+**runc features**
+
+# DESCRIPTION
+The **features** command shows the implemented features in JSON format. Features are properties of runc, such as the minimum and maximum accepted OCI versions. The implemented features may not always be available, depending on the kernel version, system libraries, CPU architecture, etc. For more information about features, you can check the features section in the runtime-spec on GitHub.
+
+# PROPERTIES
```

Comment:
A man header with such name is not standard. Please see man-pages(7) for which sections are recommended. In general, I think, what's below is part of description and doesn't require a separate section.